pkg/apis/dmi/services.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2022 The KubeEdge Authors.
```

Comment:
according to the above, @wackxu  we  can delete this file `pkg/apis/dmi/services.go` directly

Reply:
we have discussed today. we can keep this file to help developers to understand DMI.